src/components/Sidebar/HTMLIcon.jsx

```diff
@@ -0,0 +1,25 @@
+import { PureComponent } from 'react';
+import { number } from 'prop-types';
+
+export default class JavaIcon extends PureComponent {
+  static propTypes = {
+    size: number,
```

Comment:
Add a `fill` prop type and have it default to `currentColor`. When a language is selected, I expect the icon color to change (it's currently not).